kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/KSPConfig.kt

```diff
@@ -130,7 +121,7 @@ class KSPJvmConfig(
     allWarningsAsErrors,
     mapAnnotationArgumentsInJava,
 ) {
-    class Builder : KSPConfig.Builder() {
+    class Builder : KSPConfig.Builder(), Serializable {
```

Comment:
May I know why `Builder` needs to be serializable too?

Reply:
Removed.